### ISSUE TITLE
New Password field from Account Details breaks in Chrome

### ISSUE THREAD

**Issue by slavicd** [woocommerce/storefront#1289]
## Describe the bug
Inputting a "weak" password under the new password field will bring in the hint and the red, frowning emoticon telling you that the password isn't strong enough. This will offset the button that toggles plain-text/hidden modes for password to a totally inappropriate place.

Isolating the problem (mark completed items with an [x]):
- [x] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x] I can reproduce this bug consistently using the steps below.

## To Reproduce
1. create an customer account with woocommerce (or other way) 
2. go to my account page, then account details
3. try to input a new password that isn't strong enough

## Screenshots
Attached [webm](https://github.com/woocommerce/storefront/files/4307229/wc_edit_account_bug.zip) (as zip) with screencast.

### Browser Environment
Latest Chromium (77) and Chrome on Fedora

### WordPress Environment
Current Wordpress (5.3.2)

<details>
```
### WordPress Environment ###

WordPress address (URL): http://cab2b.localhost
Site address (URL): http://cab2b.localhost
WC Version: 3.9.3
REST API Version: ✔ 1.0.7
Log Directory Writable: ✔
WP Version: 5.3.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.16.1
PHP Version: 7.3.15
PHP Post Max Size: 8 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.66.0
OpenSSL/1.1.1d-fips

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.22-MariaDB
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Your server does not have the SoapClient class enabled - some gateway plugins which use SOAP may not work as expected.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.9.3
WC Database Prefix: wp_
Total Database Size: 4.52MB
Database Data Size: 3.15MB
Database Index Size: 1.37MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_email_log: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 2.02MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_posts: Data: 0.11MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_um_metadata: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_usermeta: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 18
customize_changeset: 2
nav_menu_item: 3
page: 19
post: 2
product: 15
product_variation: 11
revision: 25
scheduled-action: 9
shop_order: 1
um_directory: 1
um_form: 4

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (3) ###

Email Log: by Sudar – 2.3.2
WooCommerce Admin: by WooCommerce – 1.0.0
WooCommerce: by Automattic – 3.9.3

### Inactive Plugins (3) ###

Cabotta B2B: by Slavic Dragovtev – 0.1.0
Profile Builder: by Cozmoslabs – 3.1.2
Ultimate Member: by Ultimate Member – 2.1.4

### Settings ###

API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #6 - /
Cart: #7 - /cart/
Checkout: #8 - /checkout/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 2.5.5
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 9
Oldest: 2020-02-27 15:45:53 +0200
Newest: 2020-03-07 17:58:07 +0200

Pending: 0
Oldest: –
Newest: –

Canceled: 0
Oldest: –
Newest: –

In-progress: 0
Oldest: –
Newest: –

Failed: 0
Oldest: –
Newest: –
```
</details>

**Comment by slavicd** [comment]
Problem seems to be that code's trying to `position:absolute` inside an inline element (span).

**Comment by haszari** [comment]
I reproduced this issue in Chrome and Safari on macOS. 

The reveal password toggle ("eye" icon) overlaps with the password hint text.

<img width="755" alt="Screen Shot 2020-03-24 at 4 08 30 PM" src="https://user-images.githubusercontent.com/4167300/77384225-b766da00-6de9-11ea-9f10-354d23a29a34.png">

**Comment by haszari** [comment]
PR - https://github.com/woocommerce/storefront/pull/1325

**Comment by Aljullu** [comment]
The fix for this issue will be included in Storefront 2.5.6, which is planned to be released in a couple of weeks. If you want, you can already try the Release Candidate:
https://github.com/woocommerce/storefront/releases/tag/version%2F2.5.6-rc.1
We did a lot of internal testing, but if you find an issue, we would appreciate if you could report it.
